src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -122,6 +123,17 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 
 		this._register(this._onAddConnectionProfile);
 		this._register(this._onDeleteConnectionProfile);
+
+		const seconds = 600;
```

Comment:
It'd be nice to give this a better name (what is it for? Just the unit isn't super clear) and an explanation of how you arrived at this being the value to use